File: http/typescript/fetch/src/kiotaClientFactory.ts
import { Middleware } from "./middleware";

/**
 * Gets the default middlewares in use for the client.
 * @returns the default middlewares.
 */
export function getDefaultMiddlewares(): Middleware[] {
	return []; //TODO add default middlewares
}
/**
 * Gets the default request settings to be used for the client.
 * @returns the default request settings.
 */
export function getDefaultRequestSettings(): RequestInit {
	return {}; //TODO add default request settings
}

File: http/python/requests/http/requests/kiota_client_factory.py
from typing import Dict, List
from .middleware import Middleware

def get_default_middlewares() -> List[Middleware]:
    """Gets the default middlewares in use for the client.

    Returns:
        List[Middleware]: the default middlewares.
    """
    return []

def get_default_request_setting() -> Dict:
    """Gets the default request settings to be used for the client.

    Returns:
        RequestInit: the default request settings.
    """
    return {}